File: backend/src/group/dto/groupInfos.dto.ts
export type GroupInfos = {
    id: number,
    name: string,
    avatar: string,
    description?: string,
    creationDate: Date,
    isGroupPrivate: boolean,
    nbMembers: number,
    nbPosts: number,
    isMember: boolean,
    isAdmin: boolean
}

File: frontend/app/src/main/java/com/example/myapplication/dto/group/GroupInfosDto.kt
package com.example.myapplication.dto.group

data class GroupCardInfosDto(
    val id: Int,
    val name: String,
    val avatar: String,
    val description: String? = null,
    val creationDate: String,
    val isGroupPrivate: Boolean,
    val nbMembers: Int,
    val nbPosts: Int,
    val isAdmin: Boolean,
    val isFollowing: Boolean? = null
)
